Need to specify what group types will provision into Slack

dakoer opened the pull request:
Need to specify what groups are allowed to provision into Slack i.e. Security, mail-enabled security, Distro list, Office 365 group, etc

azurecla commented:
Hi __@gopitk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

MonicaRush commented:
@garyericson 
Gary -- Here's an ML contribution from the community. Can you let me know if this is OK to merge? Thanks a lot -- Monica

bradsev commented:
I have reviewed these edits and they are OK to merge. -Brad

dakoer committed:
Need to specify what group types will provision into Slack

Need to specify what groups are allowed to provision into Slack i.e. Security, mail-enabled security, Distro list, Office 365 group, etc
